sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs

```diff
@@ -15,5 +15,5 @@
 [assembly: AssemblyCulture("")]
 
 
-[assembly: AssemblyVersion("1.4.0.0")]
-[assembly: AssemblyFileVersion("1.4.0.0")]
+[assembly: AssemblyVersion("1.5.0.0")]
+[assembly: AssemblyFileVersion("1.5.0.0")]
```

Comment:
You don't need to bump the `AsemblyVersion` if it's just a minor upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

Reply:
Updated

Reply:
I've updated the versions. Actually for minor upgrade, bump `AssemblyFileVersion`, but don't bump `AssemblyVersion`. See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec

> 6. Update the following information in AssemblyInfo.cs
      - AssemblyFileVersion (Must have)
      - AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed